bot.rb

```diff
@@ -0,0 +1,56 @@
+require 'telegram/bot'
+require 'csv'
+require 'rutracker_api'
+
+USERNAME = ''
```

Comment:
will this code work? or you will need to put creds here before launch? :)

use .env file with dotenv gem